.gitignore

```diff
@@ -151,7 +152,7 @@ client/out/
 client/build
 
 # misc
-client/*.pem
+*.pem
```

Comment:
Move the misc to above the comment `# Client`.

Reply:
Resolved in a Slack conversation.

Reply:
This will be adjusted in a separate PR, thus resolving this conversation.